doc/source/spelling_wordlist.txt

```diff
@@ -1396,6 +1396,7 @@ InfoFormat
 InfoOptions
 Informatisé
 Informix
+InfraRed
```

Comment:
Is there any reason for the upper-case R? infrared is a word in English https://www.collinsdictionary.com/dictionary/english/infrared

Reply:
> Is there any reason for the upper-case R? infrared is a word in English

I'm just fixing sphinx-spelling warnings